packages/react/src/components/Slider/useSlider.ts

```diff
@@ -385,14 +392,14 @@ export const useSlider = (props: ISliderProps, ref: React.Ref<HTMLDivElement>) =
     ...divButtonProps,
   };
 
-  const sliderProps = {
+  const sliderProps: IFocusableElement = {
     'aria-disabled': disabled,
     role: 'slider',
     tabIndex: disabled ? undefined : 0,
     'data-is-focusable': !disabled,
   };
```

Comment:
Can you remove the IFocusableElement type and do this instead? Unfortunately TS < 4.1 doesn't provide a way to express the fact that any arbitrary data-* props are allowed on HTML elements, so we have to cast. 
```suggestion
  const sliderProps: React.HTMLAttributes<HTMLElement> = {
    'aria-disabled': disabled,
    role: 'slider',
    tabIndex: disabled ? undefined : 0,
    ...({ 'data-is-focusable': !disabled } as any)
  };
```